src/test/java/Homework19.java

```diff
@@ -9,34 +9,34 @@
 
 public class Homework19 extends BaseTest {
 
-    @Test
-    public void DeletePlaylistTest() throws InterruptedException {
-        String playlistName = "test playlist";
-
-        login("demo@class.com", "te$t$tudent");
-        wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//i[@title='Create a new playlist']"))).click();
-
-        WebElement newPlaylist = driver.findElement(By.xpath("//li[text()='New Playlist']"));
-        newPlaylist.click();
-
-        WebElement nameField = driver.findElement(By.xpath("//input[@name='name']"));
-        nameField.clear();
-        nameField.sendKeys(playlistName, Keys.ENTER);
-
-        WebElement testPlaylist = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//li/a[text()='"+playlistName+"']")));
-        testPlaylist.click();
-
-        WebElement deletePlaylist = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//button[@title='Delete this playlist']")));
-        deletePlaylist.click();
-
-        Thread.sleep(2000);
-        List<WebElement> playlistNames = driver.findElements(By.xpath("//section[@id='playlists']//li/a"));
-
-        for(WebElement p : playlistNames) {
-            String name = p.getText();
-            if (name.equals(playlistName)) {
-                Assert.assertTrue(false);
-            }
-        }
-    }
+//    @Test
+//    public void DeletePlaylistTest() throws InterruptedException {
+//        String playlistName = "test playlist";
+//
+//        login("demo@class.com", "te$t$tudent");
+//        wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//i[@title='Create a new playlist']"))).click();
+//
+//        WebElement newPlaylist = driver.findElement(By.xpath("//li[text()='New Playlist']"));
+//        newPlaylist.click();
+//
+//        WebElement nameField = driver.findElement(By.xpath("//input[@name='name']"));
+//        nameField.clear();
+//        nameField.sendKeys(playlistName, Keys.ENTER);
+//
+//        WebElement testPlaylist = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//li/a[text()='"+playlistName+"']")));
+//        testPlaylist.click();
+//
+//        WebElement deletePlaylist = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//button[@title='Delete this playlist']")));
+//        deletePlaylist.click();
+//
+//        Thread.sleep(2000);
```

Comment:
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.